docs/tutorials/gluon/save_load_params.md

```diff
@@ -61,7 +61,7 @@ def build_lenet(net):
         net.add(gluon.nn.Dense(512, activation="relu"))
```

Comment:
L13 there is also "`load_checkpoint` and `load` methods" -> "`imports` method

Reply:
we can make this change as part of another PR to avoid another round of CI.

Reply:
sorry didnt realize this was already in.